### PR TITLE
chore: release v1.8.0

### PR DESCRIPTION
**Pull request by ShadowCurse**
## Changes
Update version number / CHANGELOG / CREDITS

Also back port https://github.com/firecracker-microvm/firecracker/commit/b83640c0f7eb286103aa133175cd1d566d858a10 with `aws-lc-sys` fix. 

## Reason
1.8 release

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following Developer
Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this
  PR.
- [ ] API changes follow the [Runbook for Firecracker API changes][2].
- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.
- [ ] New `TODO`s link to an issue.
- [ ] Commits meet
  [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

______________________________________________________________________

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -39,12 +41,14 @@ Contributors to the Firecracker repository:
...
 - Andrei Cipu <acipu@amazon.com>
 - Andrei Sandu <sandreim@amazon.com>
+- Andrew Yao <andr3wy@gmail.com>
 - Andrii Radyk <ander.ender@gmail.com>
 - andros21 <andrea.ros21@murena.io>
 - Angus McInnes <angus@amcinnes.info>
 - Arjun Ramachandrula <avrdev77@gmail.com>
 - Arun Gupta <arun.gupta@gmail.com>
 - Arunav Sanyal <Khalian@users.noreply.github.com>
+- Ashwin Ginoria <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>
```
Comment (pb8o) [inline CREDITS.md:51]: Could we fix this one with a `.mailmap` entry?

File: CREDITS.md [hunk 2]
```diff
@@ -160,14 +174,17 @@ Contributors to the Firecracker repository:
 - Mihai Stan <stanmihai17cs@gmail.com>
 - milahu <milahu@gmail.com>
 - moricho <ikeda.morito@gmail.com>
+- Muskaan Singla <ec2-user@ip-172-31-1-28.us-west-2.compute.internal>
```
Comment (pb8o) [inline CREDITS.md:177]: Same, can we fix by adding `.mailmap` entry?

File: CREDITS.md [hunk 3]
```diff
@@ -195,17 +212,21 @@ Contributors to the Firecracker repository:
 - Sam Jackson <sam@clique.app>
 - Samuel Knox <knoxsa@oregonstate.edu>
 - Samuel Ortiz <sameo@linux.intel.com>
+- ScmTble <ScmTble@qq.com>
 - Sean Lavine <freewil@users.noreply.github.com>
 - Sebastien Boeuf <sebastien.boeuf@intel.com>
 - Serban Iorga <seriorga@amazon.com>
+- ShadowCurse <yegorlz@amazon.co.uk>
```
Comment (pb8o) [inline CREDITS.md:219]: Also seems can be de-duplicated ;)

File: CREDITS.md [hunk 4]
```diff
@@ -195,17 +212,21 @@ Contributors to the Firecracker repository:
...
 - shakram02 <ahmedhamdyau@gmail.com>
 - Shen Jiale <shenjiale@baidu.com>
 - Shion Yamashita <shioyama1118@gmail.com>
 - singwm <singwm@amazon.com>
+- sladynnunes <snunes@usc.edu>
 - Sripracha <ramsri@amazon.com>
 - Stefan Nita <32079871+stefannita01@users.noreply.github.com>
 - StemCll <lydjotj6f@mozmail.com>
 - Sudan Landge <sudanl@amazon.com>
+- sudanl0 <119602619+sudanl0@users.noreply.github.com>
```
Comment (pb8o) [inline CREDITS.md:229]: Here too.

File: CREDITS.md (hunk 1 above)
Comment (ShadowCurse) [reply CREDITS.md:51]: How do I fix this if it is the only entry?

File: CREDITS.md (hunk 3 above)
Comment (ShadowCurse) [reply CREDITS.md:219]: Names are different, so I don't think this can be fixed.

Comment (pb8o) [reply CREDITS.md:219]: But the email is the same. You just need an entry in mailmap like this:

```
Egor Lazarchuk <yegorlz@amazon.co.uk>
```

Comment (ShadowCurse) [reply CREDITS.md:219]: oh, ok will fix

File: CREDITS.md (hunk 4 above)
Comment (pb8o) [reply CREDITS.md:229]: For this one we would need an additional entry for Sudan:

```
Sudan Landge <sudanl@amazon.com> <119602619+sudanl0@users.noreply.github.com>
```

File: CREDITS.md (hunk 1 above)
Comment (ShadowCurse) [reply CREDITS.md:51]: 1. How did you find the email?
2. Should I add the `Ashwin Ginoria <aginoria@amazon.com>` in CREDITS and `Ashwin Ginoria <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>` in `.mailmap` or smth else?

Comment (pb8o) [reply CREDITS.md:51]: Add the line above line exactly to `.mailmap`, and re-run `update-credits.sh`

Comment (ShadowCurse) [reply CREDITS.md]: done

File: CREDITS.md (hunk 2 above)
Comment (ShadowCurse) [reply CREDITS.md]: fixed

File: CREDITS.md (hunk 3 above)
Comment (ShadowCurse) [reply CREDITS.md]: fixed

File: CREDITS.md (hunk 4 above)
Comment (ShadowCurse) [reply CREDITS.md]: fixed

File: .mailmap [hunk 5]
```diff
@@ -27,3 +27,8 @@ Trăistaru Andrei Cristian <atc@amazon.com>
 Trăistaru Andrei Cristian <atc@amazon.com> <56828222+andreitraistaru@users.noreply.github.com>
 Sudan Landge <sudanl@amazon.com> <sudanl@amazon.co.uk>
 Takahiro Itazuri <itazur@amazon.com> <zulinx86@gmail.com>
+Jack Thomson <jackabt@amazon.com> <jackabt@amazon.co.uk>
+Ashwin Ginoria <aginoria@amazon.com> <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>
+Muskaan Singla <msinglaa@amazon.com> <ec2-user@ip-172-31-1-28.us-west-2.compute.internal>
+Egor Lazarchuk <yegorlz@amazon.co.uk>
+Sudan Landge <sudanl@amazon.com> <119602619+sudanl0@users.noreply.github.com>
```
Comment (pb8o) [inline .mailmap:34]: nit: could we group Sudan's entries together? I think there are 3, but I am not sure the 1st one is needed.

**Review by pb8o — APPROVED** [review]

File: .mailmap (hunk 5 above)
Comment (ShadowCurse) [reply .mailmap]: fixed

File: CREDITS.md [hunk 6]
```diff
@@ -228,14 +247,16 @@ Contributors to the Firecracker repository:
 - William Hammond <william.t.hammond@gmail.com>
 - wllenyj <wllenyj@linux.alibaba.com>
 - wt-l00 <ei13suke@gmail.com>
+- Xiangpeng Hao <haoxiangpeng123@gmail.com>
 - xibz <impactbchang@gmail.com>
 - xiekeyang <keyang.xie@gmail.com>
 - Ye Sijun <junnplus@gmail.com>
 - Yichen Zhang <jakezhang2020@gmail.com>
+- Yılmaz ŞEN <yilmazsen94@gmail.com>
```
Comment (pb8o) [inline CREDITS.md:255]: Strange. This sorts after `Yu` with C.UTF-8 locale.

**Review by pb8o — APPROVED** [review]

**Review by kalyazin — APPROVED** [review]

**Pull request merged**